lib/cli.js

```diff
@@ -100,13 +97,25 @@ const onLoad = async (targetDir, version, argv) => {
     }
     lang = langChoice || "";
   }
+  let { jquery } = await prompts({
+    type: "confirm",
+    name: "jquery",
+    message: "Remove jQuery?",
```

Comment:
Don't we need to limit this to versions 7.3 and below?

Reply:
We didnt decide if we will remove jQuery in <7.3 or add jQuery in 8+ (or both)?

Reply:
I think it would be two different questions. I'm unsure if I want to add the option to add jQuery in 8+, but if we were to offer the option to add it, it would be different language ("would you like to add jQuery?") Removing jQuery can only happen if it's in the project. So this question should only be asked for older versions.

Reply:
That's what I meant in my question.
Ok, I'll finish in this pr the functionality for removing jQuery in <7.3 and open discussion about adding jQuery in v8+